reducers/cyStyle.js

```diff
@@ -1,8 +1,14 @@
 import DefaultVisualStyle from '../css/DefaultVisualStyle'
 
 const cyStyle = (state = DefaultVisualStyle, action) => {
-  // TODO
   switch(action.type) {
+    case "UPDATE_INTENT_LABEL":
+      return state.map(s => {
+        if (s.selector == 'node') {
+          s.css.label = 'data(' + action.value + ')';
```

Comment:
Every s in the state is modified here. If you only want to modify some part of the values in s, you can create a new object and then modify it by using Object.assign(). Please refer to the  "ADD_INTENT" case in cyElements.js.